src/engine/renderer/tr_shader.cpp

```diff
@@ -5572,6 +5572,88 @@ bool CheckShaderNameLength( const char* func_err, const char* name, const char*
 	return true;
 }
 
+static bool CheckStage( shaderStage_t *pStage )
```

Comment:
This could take a const stage. Caller should mark it not active

Reply:
I can make the caller to mark it inactive but this is remaining:

```
tr_shader.cpp:5624:39: error: cannot assign to variable 'pStage'
with const-qualified type 'const shaderStage_t *'
 5624 |                             pStage->bundle[ 0 ].image[ 0 ]  = tr.defaultImage;
      |                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ^
```